added more detail to the quantity to be deducted field (#883)
magento/merchdocs: src/catalog/inventory-shipment-create.md

   - For  **Select a Source to Ship from**, select a source from the menu to send a shipment.

>    - For each line item, keep the recommended amount or enter a specific amount in the **Qty to Deduct** which deducts the entered amount from the inventory of the selected source.

Reviewer: ```suggestion
   - For each line item, keep the recommended amount or enter a specific amount in the **Qty to Deduct**. This is the amount that is deducted from the inventory of the selected source.
```
